src/box/lua/schema.lua

```diff
@@ -2525,6 +2525,10 @@ end
 -- the same tuple as index:pairs(key, {iterator = it})
 base_index_mt.offset_of = function(index, key, opts)
     check_index_arg(index, 'offset_of', 2)
+    key = keify(key)
+    if opts == nil then
+        opts = {}
+    end
```

Comment:
What's about invalid `opts`? Let's use `check_param_table` helper.

Reply:
Introduced the invalid option check.